### ISSUE TITLE
Invalid API calls, even when plugin not being used by WooCommerce

### ISSUE THREAD

**Issue by whitehawkJulie** [woocommerce/woocommerce-paypal-payments#2024]
### Describe the Bug

When I activate the plugin, my website slows down horrifically (and/or 504's my site), and I get a bunch of errors as shown in the image. It can take minutes to load a single back end page. If it doesn't 504, Query Monitor shows me these:

<img width="1380" alt="Screenshot 2024-02-10 at 8 13 24 pm" src="https://github.com/woocommerce/woocommerce-paypal-payments/assets/10231258/fd093280-2f56-460f-930a-3f5212163198">

Here's the full stack trace of one of the failing calls, if that helps:

<img width="561" alt="Screenshot 2024-02-10 at 10 08 05 pm" src="https://github.com/woocommerce/woocommerce-paypal-payments/assets/10231258/55272f20-e778-4a50-9549-9d9e0b69230a">

It even happens when I first activate the plugin - it's not even set up with WooCommerce yet, so shouldn't even be being called yet, surely??

### To Reproduce

1. WooCommerce is installed. There are no rows in wp_woocommerce_payment_tokens or ..meta tables.
2. Activate plugin
3. Try to go to the plugin's settings, wait for several minutes
4. 504 - Gateway Timeout ... or if it loads, then the kinds of errors in screenshot above.

### Environment

 - WordPress 4.6.3
 - WooCommerce 8.5.2
 - this Plugin 2.5.3
 - Browser - Firefox v122.0 
 - Any other plugins installed - NONE!
 - Theme - changed to 2023, still getting same error

I have no caching plugins, and turned off all server level caching, and that didn't make a difference (as per [this](https://wordpress.org/support/topic/authentication-with-paypal-failed-could-not-create-token/) which seemed similar to my issue)

**Comment by whitehawkJulie** [comment]
I've made a local copy, with ONLY Woocommerce and WC Paypal Payments installed, and changed the theme, and I'm STILL getting the same issue. I don't get the issue if I install them both in a clean wordpress installation. I'm totally out of ideas in how to debug this!

**Comment by Sirvijver** [comment]
Hello @whitehawkJulie 

You have reached out to open GitHub forum with a support request. We utilize GitHub mainly for development activities, so I would like to ask you to [reach out to the support team](https://woocommerce.com/document/woocommerce-paypal-payments/#get-help) directly for more guidance.

This looks a lot like this issue: https://github.com/woocommerce/woocommerce-paypal-payments/issues/1816
We have got a thread open for this, but since it does not have a direct impact (failing orders or critical site failure), the priority is lower for such cases. I have added this report to the bug and asked the team if this can be investigated.
It might be good to open a private support thread with us, so we can more easily request for additional details on your site and the issue (logs), instead of communicating via this public forum. I would advise to at least follow the related GH issue, since we use that one as the main GH thread for development if the dev team needs to make any update to the plugin for this issue.
I would advise you to see if onboarding your account with the connect wizard might help resolve the issue, and possibly to reach out to [PayPal Merchant Technical Support](https://www.paypal-techsupport.com/s/contactsupport) to have them have a look at the backend to see why these requests are being sent. They have access to verify the backend; we don't.

I'll mark this thread as resolved, but leave the first GH thread open for further dev activities in case required.

Kind regards, 
Joost
